server/views/includes/index-poetry.pug

```diff
@@ -0,0 +1,139 @@
+.row 
+    .col
+        div(id="carouselExampleControls" class="carousel slide" data-ride="carousel")
```

Comment:
Should example still be in this id name? :)

Reply:
Updated the ID name to not include "example"